iothub/service/tests/ClientWebSocketTransportTests.cs

```diff
@@ -79,6 +79,7 @@ public async Task ClientWebSocketTransportReadWithoutConnectTest()
 
         // The following tests can only be run in Administrator mode
         [TestMethod]
+        [Timeout(10000)] // if it is going to fail, fail fast. Otherwise it can go on for 4+ minutes. :(
```

Comment:
Just curious - why does this fail?

Reply:
The comment on 80 says it has to be run in admin mode.

In practice, I find it fails if run with other tests and not in admin mode. If you run it by itself, even not in admin mode, it'll succeed.